scripts/release.sh

```diff
@@ -2,6 +2,15 @@
 
 VERSION=$1
 
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+echo "Was starknet.rs package updated during this release cycle?"
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) echo "What is the spec version it supports?";read spec_version;echo $spec_version > "$parent_path/../crates/forge/expected-rpc-version";break;;
+        No ) break;;
+    esac
+done
+
```

Comment:
I don't think this is needed at all; if someone makes a change to update starkner-rs with a new spec version, and this very same person does not update said file straight away - this is a bug. Moreover, I'm pretty positive at least some of currently existing sncast tests would fail if that's the case + @ddoktorski is actually adding new tests (based on devnet) to specifically test the spec versions.

Reply:
For further clarity, we will add a simple test to ensure the devnet spec version corresponds with our currently supported version. As a result, overlooking an update to the supported RPC version will become highly unlikely + as @THenry14 said some sncast tests probably would fail anyway.

Reply:
If this will be clear enough (commented preferably alongside the test?) for the dev to deduce, then i am happy. That is the main concern - not to forget it and make it obsolete in the process